src/HotChocolate/Core/src/Types.Scalars/EmailAddressType.cs

```diff
@@ -0,0 +1,104 @@
+using HotChocolate.Language;
+using HotChocolate.Types.Scalars;
+
+namespace HotChocolate.Types
+{
+    /// <summary>
+    /// A field whose value conforms to the standard internet email address format as specified in HTML Spec
```

Comment:
I guess something like this would be better. 
Got it from the string scalar and slightly modified it:
```
The `EmailAddress` scalar type represents a email address, represented as UTF-8 character sequences. The scalar follows the specification defined in RFC 5322
```